jinahub/rankers/LightGBMRanker/lightgbm_ranker.py

```diff
@@ -0,0 +1,209 @@
+__copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+from typing import Dict, Optional, List
+
+import lightgbm
+import numpy as np
+from jina.logging.logger import JinaLogger
+from jina import Executor, DocumentArray, requests
+
+
+class LightGBMRanker(Executor):
+    """
+    Computes a relevance score for each match using a pretrained learning-to-rank model trained with LightGBM (https://lightgbm.readthedocs.io/en/latest/index.html)
+    :param query_features: name of the features to extract from query Documents and used to compute relevance scores by the model loaded
```

Comment:
add empty line for better readability
```suggestion

    :param query_features: name of the features to extract from query Documents and used to compute relevance scores by the model loaded
```